docs/migration.md

```diff
@@ -119,7 +172,7 @@ For migrating to the DOM interaction helpers, you can use the
 [ember-test-helpers-codemod](https://github.com/simonihmig/ember-test-helpers-codemod)
 to automatically convert all or most of it.
 
-### Acceptance tests
+#### Acceptance tests
```

Comment:
Should this read `Application tests (formerly Acceptance tests)` now?

Reply:
Ya, good idea